third_party/zlib/CMakeLists.txt

```
@@ -48,6 +48,10 @@ else()
             zlib/x86.c
             zlib/x86.h
         )
+
+        if(NOT MSVC)
+            set(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} -msse4.2 -mpclmul")
```

```suggestion
            target_compile_options(crashpad_zlib PRIVATE "-msse4.2 -mpclmul")
```
That should work I guess. Or does that need to be on the linker flags as well?

---

I already tried this, so it should work.
https://github.com/getsentry/crashpad/blob/4ab7d237ff66ed82bc6e01ac159f1de81dae26cf/third_party/zlib/BUILD.gn#L134-L139
This indicates that it only needs the flags for the C compiler.